frontend/amundsen_application/static/.betterer.results

```diff
@@ -148,26 +148,33 @@ exports[`eslint`] = {
       [77, 16, 50, "Static HTML elements with event handlers require a role.", "1837974012"],
       [78, 19, 21, "Must use destructuring state assignment", "3239125001"]
     ],
-    "js/components/ResourceListItem/DashboardListItem/index.spec.tsx:3836381581": [
-      [47, 14, 5, "\'props\' is assigned a value but never used.", "187023499"],
-      [63, 6, 25, "Use object destructuring.", "354229464"],
-      [64, 6, 29, "Use object destructuring.", "2645724888"],
-      [146, 18, 5, "\'props\' is assigned a value but never used.", "187023499"]
-    ],
-    "js/components/ResourceListItem/DashboardListItem/index.tsx:401041810": [
-      [17, 0, 41, "\`../types\` import should occur before import of \`./constants\`", "2620565544"]
-    ],
-    "js/components/ResourceListItem/FeatureListItem/index.spec.tsx:3348265572": [
-      [45, 6, 25, "Use object destructuring.", "354229464"],
-      [46, 6, 29, "Use object destructuring.", "2645724888"]
+    "js/components/ResourceListItem/DashboardListItem/index.spec.tsx:2412974137": [
+      [51, 14, 5, "\'props\' is assigned a value but never used.", "187023499"],
+      [67, 6, 25, "Use object destructuring.", "354229464"],
+      [68, 6, 29, "Use object destructuring.", "2645724888"],
+      [150, 18, 5, "\'props\' is assigned a value but never used.", "187023499"]
+    ],
+    "js/components/ResourceListItem/FeatureListItem/index.spec.tsx:3223391413": [
+      [49, 6, 25, "Use object destructuring.", "354229464"],
+      [50, 6, 29, "Use object destructuring.", "2645724888"]
+    ],
+    "js/components/ResourceListItem/MetadataHighlightList/highlightingUtils.ts:2480902327": [
+      [29, 2, 19, "Assignment to function parameter \'originalDescription\'.", "1982058394"],
+      [30, 2, 22, "Assignment to function parameter \'highlightedDescription\'.", "2370485106"]
+    ],
+    "js/components/ResourceListItem/MetadataHighlightList/index.spec.tsx:1748169974": [
+      [6, 0, 67, "\`./highlightingUtils\` import should occur before import of \`.\`", "2024311329"],
+      [7, 0, 48, "\`components/SVGIcons\` import should occur before import of \`.\`", "2948929935"],
+      [29, 10, 25, "Use object destructuring.", "354229464"],
+      [30, 10, 29, "Use object destructuring.", "2645724888"]
```

Comment:
Let's make sure we don't add any new items on this list, happy to help on finding alternative syntaxes.